controllers/ProfileController.js

```diff
@@ -0,0 +1,58 @@
+const pool = require("../database/db");
+const profileQueries = require("../models/ProfileModel");
+
+const getOwnTweets = async (req, res) => {
+  try {
+    const { userId } = req.query;
+    const query = await pool.query(profileQueries.getOwnTweets, [userId]);
+    res.send(query.rows);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send(error);
+  }
+};
+
+const getOwnReplies = async (req, res) => {
+  try {
+    const { userId } = req.query;
+    const query = await pool.query(profileQueries.getOwnReplies, [userId]);
+    res.send(query.rows);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send(error);
+  }
+};
+
+const getOwnLikes = async (req, res) => {
+  try {
+    const { userId } = req.query;
+    const query = await pool.query(profileQueries.getOwnLikes, [userId]);
+    res.send(query.rows);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send(error);
+  }
+};
+
+const getProfileContents = async (req, res) => {
+  try {
+    const { userId } = req.query;
+    const query = await pool.query(profileQueries.getProfileContents, [userId]);
+    const date = new Date(query.rows[0].joindate);
+    const month = date.toLocaleString("default", { month: "long" });
+    const year = date.getFullYear();
+    const formattedDate = `${month} ${year}`;
+    query.rows[0].joindate = formattedDate;
```

Comment:
This feels kinda gross to me cause we didn't extract the contents of `query` out into a variable.. It looks weird cause we declared `query` to be a `const` variable on line 40, so it it looks like we're modifying a `const`. Can we maybe try something like this instead?

```suggestion
    const profileContents = query.rows[0];
    res.send({...profileContents, joinedDate: formattedDate});
```